Content.Server/Cargo/CargoOrderDatabase.cs

```diff
@@ -86,9 +90,16 @@ public bool RemoveOrder(int orderNumber)
         /// <param name="order">The order to be approved.</param>
         public void ApproveOrder(int orderNumber)
         {
+            if (CurrentOrderSize == MaxOrderSize)
+                return;
             if (!_orders.TryGetValue(orderNumber, out var order))
                 return;
+            else if (CurrentOrderSize + order.Amount > MaxOrderSize) { 
+                AddOrder(order.Requester, order.Reason+" (Overflow)", order.ProductId, order.Amount - MaxOrderSize - CurrentOrderSize, order.PayingAccountId);
```

Comment:
Use string interpolation for the second argument:

```csharp
AddOrder(order.Requester, $"{order.Reason} (Overflow)", ...
```

Reply:
Actually, use localized strings instead:

```cs
AddOrder(order.Requester, Loc.GetString("{0} (Overflow)", order.Reason), ...
```